src/pointer-analysis/value_set_dereference.cpp

```diff
@@ -61,6 +61,43 @@ static bool should_use_local_definition_for(const exprt &expr)
   return false;
 }
 
+static void do_display_points_to_sets(
```

Comment:
Minor quibble about the naming :-) Its not the most descriptive name in the world, so perhaps something like `display_points_to_sets_json`. No major preference though, so just take it as a suggestion but nothing more than that.